src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,92 @@
+import classNames from 'classnames';
+import React, { useCallback } from 'react';
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { PersonLink } from '../PersonLink';
+
+type Props = {
+  people: Person[],
+  loading: boolean,
+  isError: boolean,
+};
+
+export const PeopleTable: React.FC<Props> = ({
+  people,
+  loading,
+  isError,
+}) => {
+  const { personData = '' } = useParams();
+
+  const findParent = useCallback((parentName: string | null) => {
+    if (!parentName) {
+      return '-';
+    }
+
+    const possibleParent = people.find(person => person.name === parentName);
+
+    return possibleParent
+      ? <PersonLink person={possibleParent} />
+      : parentName;
+  }, [people]);
+
+  return (
+    <div className="block">
+      <div className="box table-container">
+        {loading
+          ? <Loader />
+          : (isError && (
+            <p data-cy="peopleLoadingError" className="has-text-danger">
+              Something went wrong
+            </p>
+          )) || ((people.length === 0) && (
```

Comment:
Don't nest everything in one ternary operator. Your code looks a bit complicated. 
Just provide separate checks